pkg/operation/etcdencryption/etcdencryptioninfodata.go

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2020 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+ * Copyright 2020 SAP SE or an SAP affiliate company and Gardener contributors
```

Comment:
missed by replace due to different formatting

Reply:
Hm.. what else was missed? TBH, this PR is pretty much not reviewable, simply too large. It was just luck that I spotted these few instances, but maybe there are even more. Doesn't it make rather sense to introduce some validation script to be sure no files are being missed?

Reply:
+1 maybe in `make verify` that would also be helpful for the future.

Reply:
After you found these issues I reviewed the change using git diff locally (what a PITA).
Also looked at git log --numstat to spot files which stick out.

I am not sure how a script validating this change should look like since the existing license headers on the master branch seem not to be fully consistent. Any ideas ?

We could also completely remove license headers and only rely on the annotations in .reuse/dep5 which is allowed
by the reuse spec. In the first round I only did that for the vendored dependencies but not for the project source code since I think that this would increase likelihood that someone who copies some of the source code looses track of the provenance, license and copyright of the copied files if he doesn't look at the annotations in .reuse/dep5.
A change removing all license headers would have a similar risk and validating
it seems to be of similar complexity like adapting the license headers to the new rules.

I used a couple of shell scripts and https://github.com/go-enry/go-license-detector to prepare this change.
Also pushed similar changes for apiserver-proxy and autoscaler. With the rudimentary tooling I used to prepare
these changes this involves too much manual work for creating the metadata in .reuse/dep5 for the vendored
dependencies. In order to do this on all 80 Gardener repositories and keep it up-to-date when dependencies change we need better tooling.

Reply:
> I am not sure how a script validating this change should look like since the existing license headers on the master branch seem not to be fully consistent. Any ideas ?

An easy first shot could be to check if the files contain the

```
SPDX-FileCopyrightText: 2020 SAP SE or an SAP affiliate company and Gardener contributors 
SPDX-License-Identifier: Apache-2.0
```

(year should not matter/only start with `20`). WDYT?